src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                    "description": "Version der Corona-Warn-App"
+                },
+                {
+                    "term": "Risikobegegnung",
+                    "anchor": "risk_encounter",
+                    "description": "aufgezeichnete Begegnung mit einer Person, die einen selbst möglicherweise aufgrund von Dauer und Abstand zur Infektion geführt haben könnte"
+                },
+                {
+                    "term": "Risikoberechnung",
+                    "anchor": "risk_calculation",
+                    "description": "Vier Faktoren haben einen Einfluss auf die Berechnung des individuellen Infektionsrisikos:<ul><li>Dauer der Begegnung</li><li>Abstand bei der Begegnung</li><li>Zeit, die seit der Begegnung vergangen ist</li><li>Infektiösität (abgeleitet aus der Information, wann und ob die Symptome eingesetzt haben)</li></ul> Überschreitet der daraus ermittelte Risikowert den vom Robert Koch-Institut vorgegebenen Schwellwert, erhalten die Benutzer eine Benachrichtigung über ein erhöhtes Infektionsrisiko."
+                },
+                {
+                    "term": "Risikostatus",
+                    "anchor": "risk_status",
+                    "description": "Eine Kachel in der Corona-Warn-App, die mit ihrer Farbe und dem Inhalt die eigene Gefährdung vor einer Corona-Infektion darstellt. Der Status setzt sich aus den Begegnungen, die man mit anderen Personen gemacht hat, zusammen. Bei einem niedrigen Infektionsrisiko ist die genannte Kachel grün, bei einem erhöhten Risiko ist die Kachel rot."
+                },
+                {
+                    "term": "Rolling Proximity Identifier (RPI)",
+                    "anchor": "rpi",
+                    "description": "IDs, die den Datenschutz sicherstellen und sich regelmäßig in kurzen Abständen ändern und so eine Nachverfolgung des Geräts verhindern. Diese IDs können lediglich mit ebenfalls <a href='#glossary_pseudonymity'>pseudonymen</a> <a href='#glossary_diagnosis_key'>Diagnoseschlüsseln</a> ('Positivkennungen') verknüpft werden, um die Begegnungserkennung durchzuführen. Da sich diese IDs alle 10 bis 20 Minuten ändern, könnte z. B. eine Person, die jeden Tag im Bus neben Ihnen sitzt, Sie selbst nicht mit einer ID in Verbindung bringen, wenn die Liste der verfügbaren RPIs einsehbar wäre."
                 }
             ],
             "S": [
+                {
+                    "term": "SARS-CoV-2",
+                    "anchor": "sars_cov_2",
+                    "description": "Siehe <a href='#glossary_corona_virus'>Corona-Virus</a>"
+                },
+                {
+                    "term": "Science-Blog",
+                    "anchor": "science_blog",
+                    "description": "Im Science-Blog der Corona-Warn-App werden verschiedene wissenschaftliche Themen zur Corona-Warn-App wie beispielsweise die Wirksamkeit, den Nutzen und den Nutzern sowie Ergebnisse wissenschaftlicher Evaluationen des Robert Koch-Instituts behandelt."

```

Comment:
'und den Nutzern' ist überflüssig

Reply:
Will be changed in the next commit, thanks!